src/app/routes/getInitialData/frontpage/index.test.js

```diff
@@ -4,10 +4,12 @@ import fetchData from '../utils/fetchData';
 import filterUnknownContentTypes from '../../../lib/utilities/preprocessor/rules/filterContentType';
 import filterEmptyGroupItems from '../../../lib/utilities/preprocessor/rules/filterEmptyGroupItems';
 import applySquashTopstories from '../../../lib/utilities/preprocessor/rules/topstories';
+import addIdsToItems from '../../../lib/utilities/preprocessor/rules/addIdsToItems';
 import filterGroupsWithoutStraplines from '../../../lib/utilities/preprocessor/rules/filterGroupsWithoutStraplines';
 
 const preprocessorRules = [
   filterUnknownContentTypes,
+  addIdsToItems,
```

Comment:
Should this come after `filterEmptyGroupItems`

Reply:
Why? - I'm assuming the reason is because the rules will be applied to jsonRaw sequentially by the reduce function in `src/app/lib/utilities/preprocessor/index.js` - but then the issue of what sequence should be used comes up?

Reply:
Went ahead and changed the sequence